apps/web/src/stores/evaluationResultCharts/numericalResults/averageResultAndCostOverCommitStore.ts

```diff
@@ -1,35 +1,35 @@
+import { useCallback } from 'react'
+
 import { Evaluation } from '@latitude-data/core/browser'
 import { computeAverageResultAndCostOverCommitAction } from '$/actions/evaluationResults/computeAggregatedResults'
-import useSWR, { SWRConfiguration } from 'swr'
+import useSWR from 'swr'
 
-export default function useAverageResultsAndCostOverCommit(
-  {
-    evaluation,
-    documentUuid,
-  }: {
-    evaluation: Evaluation
-    documentUuid: string
-  },
-  opts: SWRConfiguration = {},
-) {
-  const { data, isValidating, isLoading, error } = useSWR(
-    ['averageResultAndCostOverCommit', evaluation.id, documentUuid],
-    async () => {
-      const [data, error] = await computeAverageResultAndCostOverCommitAction({
-        documentUuid,
-        evaluationId: evaluation.id,
-      })
+export default function useAverageResultsAndCostOverCommit({
+  evaluation,
+  documentUuid,
+}: {
+  evaluation: Evaluation
+  documentUuid: string
+}) {
+  const fetcher = useCallback(async () => {
+    const [data, error] = await computeAverageResultAndCostOverCommitAction({
+      documentUuid,
+      evaluationId: evaluation.id,
+    })
 
-      if (error) return []
-      return data
-    },
-    opts,
+    if (error) return []
+    return data
+  }, [documentUuid, evaluation.id])
+  const { data, isValidating, isLoading, error, mutate } = useSWR(
+    ['averageResultAndCostOverCommit', evaluation.id, documentUuid],
+    fetcher,
   )
 
   return {
     data,
     isLoading,
     isValidating,
     error,
+    refetch: mutate,
```

Comment:
yeah i wouldn't do this, just expose mutate and have devs learn what it can be used for

Reply:
Talking in the other convo https://github.com/latitude-dev/latitude-llm/pull/248#discussion_r1774656399